proposals/NNNN-noncopyable-generics.md

```diff
@@ -167,20 +179,23 @@ identity(FileDescriptor())  // OK, FileDescriptor is not Copyable
 identity([1, 2, 3])  // OK, even though Array<Int> is Copyable
 ```
 
-> **Key Idea:** Applying an inverse `T: ~Copyable` does _not_ prevent a `Copyable` type from being substituted for `T`, i.e., it does _not_ narrow the kinds of types permitted, like a constraint would. This is the reason why `~Copyable` is referred to as the _inverse_ of `Copyable` rather than a constraint itself. When applying an inverse `~C` to a type variable `T` constrained to `C`, that constraint is nullified only if it was a default requirement of `T`. Removing any kind of constraint from a generic parameter _broadens_ the kinds of types permitted.
+> **Key Idea:** Suppressing the `Copyable` requirement by using `T: ~Copyable` does _not_ prevent a `Copyable` type from being substituted for `T`.
+> This is the reason why the syntax `~Copyable` is referred to as _suppressing_ `Copyable` rather than _inverting_ or _negating_ it.
```

Comment:
The removed part here scratches at the reasoning behind why `T: ~Copyable` is not to be interpreted as "T can only accept noncopyable types".  It needs a better explanation with an example so this is fine to remove.